### ISSUE TITLE
Can't convert from pyarrow.Table

### ISSUE THREAD

**Issue by dhirschfeld** [xhochy/fletcher#98]
```python
In [319]: pa.__version__
Out[319]: '0.15.1'

In [320]: fletcher.pandas_from_arrow(tbl)
C:\Users\dhirschf\envs\dev\lib\site-packages\fletcher\base.py:731: FutureWarning: Calling .data on ChunkedArray is provided for compatibility after Column was removed, simply drop this attribute
  data[col.name] = FletcherArray(col.data)
Traceback (most recent call last):

  File "<ipython-input-320-a9ca90d82a7d>", line 1, in <module>
    fletcher.pandas_from_arrow(tbl)

  File "C:\Users\dhirschf\envs\dev\lib\site-packages\fletcher\base.py", line 731, in pandas_from_arrow
    data[col.name] = FletcherArray(col.data)

AttributeError: 'pyarrow.lib.ChunkedArray' object has no attribute 'name'


In [321]: type(tbl)
Out[321]: pyarrow.lib.Table
```

py37/win64 - fletcher=0.2.0

**Comment by jorisvandenbossche** [comment]
@dhirschfeld this has been fixed in master, so it might be best to try out the latest version (it's pure python so easy to install) until there is a new release.

**Comment by dhirschfeld** [comment]
Fixed in #86
